src/PKSim.Core/Snapshots/Mappers/SimulationMapper.cs

```diff
@@ -107,9 +107,23 @@ public override async Task<SnapshotSimulation> MapToSnapshot(ModelSimulation sim
          snapshot.IndividualAnalyses = await _simulationTimeProfileChartMapper.MapToSnapshots(simulation.AnalysesOfType<SimulationTimeProfileChart>());
          snapshot.PopulationAnalyses = await _populationAnalysisChartMapper.MapToSnapshots(simulation.AnalysesOfType<ModelPopulationAnalysisChart>());
          snapshot.HasResults = simulation.HasResults;
+         snapshot.AlteredBuildingBlocks = alteredBuildingBlocksIn(simulation);
```

Comment:
I hope when reading from old snapshot where `snapshot.AlteredBuildingBlocks` was not set - it will not crash?

Reply:
Nope. But we'll find out for sure soon enough